src/components/Sidebar.jsx

```diff
@@ -12,33 +13,62 @@ const menus = [
     icon: "fas fa-home",
     path: "/timeline"
   },
-  {
-    name: "Search",
-    icon: "fas fa-search"
-  },
-  {
-    name: "Notifications",
-    icon: "fas fa-bullhorn"
-  },
-  {
-    name: "Messages",
-    icon: "far fa-envelope"
-  },
-  {
-    name: "Bookmarks",
-    icon: "far fa-bookmark"
-  },
   {
     name: "Profile",
     icon: "far fa-user",
     path: "/profile"
   },
   {
-    name: "More",
-    icon: "fas fa-list"
+    name: "Logout",
+    icon: "fas fa-sign-out-alt",
+    handler: "out"
   }
 ];
 
+const renderMenu = (handleClick) => {
+  return menus.map(menu => (
+    <List key={menu.name}>
+      <Icon className={menu.icon}></Icon>
+      {menu.path ? (
+        <Link 
+          to={menu.path} 
+          style={{ textDecoration: "none", color: "#fff" }}
+        >
+          <Span>{menu.name}</Span>
+        </Link>
+      ) : (menu.handler) ?(
+        <Link 
+          to={menu.path} 
```

Comment:
There is a `path` item when menu has a `handler`?

Reply:
I reckon that you can just use a button element and redirect users to other URLs inside the `handleClick` function.

Reply:
I mean, we should use the button if that does not need a `href` attr

Reply:
just followed the flow of how it was made.
removed Logout from the menus object and created an independent link after the map.